### PR TITLE
chore(tracing)!: Remove deprecated install excepthook

### PR DESCRIPTION
**Pull request by mabdinur**
## Checklist
- [ ] Added to the correct milestone.
- [ ] Tests provided or description of manual testing performed is included in the code or PR.
- [ ] Library documentation is updated.
- [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).

### CODE REVIEW

File: releasenotes/notes/remove-ddtrace-install-excepthooks-8107b679a9f51ef3.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    Remove deprecated `ddtrace.install_excepthook`.
+  - |
+    Remove deprecated `ddtrace.uninstall_excepthook`.
```
Comment (brettlangdon) [inline releasenotes/notes/remove-ddtrace-install-excepthooks-8107b679a9f51ef3.yaml:6]:
```suggestion
    Remove deprecated ``ddtrace.install_excepthook``.
  - |
    Remove deprecated ``ddtrace.uninstall_excepthook``.
```

**Review by brettlangdon — APPROVED** [review]

**Review by majorgreys — APPROVED** [review]

File: ddtrace/contrib/requests/patch.py [hunk 2]
```diff
@@ -31,7 +31,6 @@ def patch():
     _w("requests", "Session.send", _wrap_send)
     Pin(_config=config.requests).onto(requests.Session)
 
-
```
Comment (brettlangdon) [inline ddtrace/contrib/requests/patch.py:34]: unrelated change?

Comment (mabdinur) [reply ddtrace/contrib/requests/patch.py:34]: Yup, it was added while fixing merge conflicts. I'll reset this branch
